doc/newsboat.asciidoc

```diff
@@ -791,8 +791,8 @@ Attribute:Context:Meaning
 [[attr-date]]<<attr-date,+date+>>:article:publication date of the article
 [[attr-guid]]<<attr-guid,+guid+>>:article:a unique identifier of the article
 [[attr-unread]]<<attr-unread,+unread+>>:article:indicates whether the article has been read
-[[attr-enclosure_url]]<<attr-enclosure_url,+enclosure_url+>>:article:the URL of a possible enclosure (e.g. podcast file)
-[[attr-enclosure_type]]<<attr-enclosure_type,+enclosure_type+>>:article:the MIME type of the enclosure URL
+[[attr-enclosure_url]]<<attr-enclosure_url,+enclosure_url+>>:article:the URL of an enclosure (e.g. podcast file). Empty if there is no enclosure
```

Comment:
I would replace the full-stop with a comma and continue in lowercase, because the first sentence does not start with an uppercase letter and the second sentence misses the full-stop. This is then consistent with the rest.

Reply:
Good idea; thanks!